<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Ключ 8 байт</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonInPath">
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEditXORKey">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Режим работы:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetTimer" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Каждые</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTimer">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>мс</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtoOutPath">
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelInPath">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Путь к входным файлам:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBoxDeleteInFiles">
      <property name="text">
       <string>Удалять входные файлы</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboBoxResultFileAct"/>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QListWidget" name="listWidgetFiles"/>
    </item>
    <item row="10" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStop">
        <property name="text">
         <string>Стоп</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStart">
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditInFileMask">
      <property name="placeholderText">
       <string>Маска входных файлов *.txt,  testFile.bin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>При наличии результирующего файла: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="labelOutPath">
      <property name="text">
       <string>Путь сохранения выходных файлов</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QLabel" name="labelComplitedFiles">
      <property name="text">
       <string>Обработанные файлы</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
